### DISCUSSION TITLE
streamdeck-linux-ui Arch AUR Repositories.

### DISCUSSION THREAD

**Discussion by coolapso** [streamdeck-linux-gui/streamdeck-linux-gui#47] [Announcements]
I have just pushed a new package to the Arch AUR Repository maintained by me. 
This package builds from the Development pre-releases and I will be updating it every time there is a relevant pre-release. 

As it of the date of writing this  currently the status of AUR packages related to streamdeck-linux-ui is the following: 

streamdeck-ui: Stable releases
streamdeck-ui-develop: Development pre-releases
streamdeck-ui-git: Using main branch, should be as stable as streamdeck-ui package, but this one builds and installs from latest commits on main branch. 

I will Update this announcement every now and then if there are any changes.

**Comment by Danie10** [comment]
Very nice thanks!

**Comment by Danie10** [comment]
OK seems to have worked for me, installing Dev version (it uninstalled stable). But I noticed:
1. Help / About Version shows 3.0.1. Did restart the app but maybe it is picking up from the config file.
2. Clicking the Help / Github took me to https://github.com/timothycrosley/streamdeck-ui so maybe that must be updated.

**Reply by coolapso** [reply]
yeah ...the Version will always show the latest release stable version as it gets it from the package config file. I haven't found a very reliable way to get the development version instead without having to keep updating the config file. 

about the 2 point, most likely that reference is still somewhere.

**Reply by coolapso** [reply]
links to old repo should be fixed now: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/commit/cfaea5800bdce9f2367aa5e1ed8571e3ebaee02e

should be available on the next pre-release.

**Comment by coolapso** [comment]
Edited, Updated info about `streamdeck-ui-git` AUR package
